sispmctl.coffee

```diff
@@ -38,37 +38,49 @@ module.exports = (env) ->
       @name = config.name
       @id = config.id
       super()
+      @_updateStatus() if @config.interval > 0
 
-    getState: () ->
-      if @_state? then return Promise.resolve @_state
-      # Built the sispmctrl command to get the outlet status
+    _updateStatus: () ->
+      setTimeout () =>
+        @_getState()
+        .finally () =>
```

Comment:
Are you sure we should be using done() here?
```
.done(
    [function(any value) fulfilledHandler],
    [function(any error) rejectedHandler]
) -> undefined
```
Like .then, but any unhandled rejection that ends up here will crash the process (in node) or be thrown as an error (in browsers). The use of this method is heavily discouraged and it only exists for historical reasons.